testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -6569,3 +6577,69 @@ fn use_latest_tip_integration_test() {
     )
     .is_ok());
 }
+
+#[test]
+#[ignore]
+fn test_flash_block_skip_tenure() {
```

Comment:
Should this test be added to the CI test matrix? Look at `.github/workflows/bitcoin-tests.yml`

Reply:
Ha, yes, forgot. Will add